stories/Display/TabsStory.tsx

```diff
@@ -31,6 +33,14 @@ const props = {
       description:
         'Sets alignment of Tab. The vertical tabs are changed to horizontal ones for screen size below 768',
     },
+    {
+      name: 'colour',
```

Comment:
I think we can rename it to `theme` to have the consistent name of the color prop because `Button` and `Divider` both are using `theme` prop.